formula-android/src/main/java/com/instacart/formula/fragment/FragmentFlowState.kt

```diff
@@ -11,10 +12,18 @@ import com.instacart.formula.integration.KeyState
  * @param states Last emitted state of each active [FragmentContract].
  */
 data class FragmentFlowState(
-    val activeKeys: List<FragmentKey> = emptyList(),
-    val visibleKeys: List<FragmentKey> = emptyList(),
-    val features: Map<FragmentKey, FeatureEvent> = emptyMap(),
-    val states: Map<FragmentKey, KeyState> = emptyMap()
+    val activeIds: List<FragmentId> = emptyList(),
+    val visibleIds: List<FragmentId> = emptyList(),
+    val features: Map<FragmentId, FeatureEvent> = emptyMap(),
+    val states: Map<FragmentId, KeyState> = emptyMap()
 ) {
-    fun visibleState() = visibleKeys.lastOrNull()?.let { states[it] }
+    @Deprecated("use activeIds")
+    val activeKeys: List<FragmentKey>
```

Comment:
Instead of breaking the API, I'm leaving `activeKeys` and `visibleKeys` for now.